src/framework/parsers/ply.js

```diff
@@ -609,6 +609,9 @@ class PlyParser {
                     }
                 );
 
+                // allow application to process the data
+                asset.fire('load:data', data);
```

Comment:
So I guess this is just under the hood undocumented call for internal use?

Reply:
SuperSplat editor will use it immediately so I guess we should be documenting. It's kind of a niche event that I don't expect developers will need...

In general I have left these asset loading details out of documentation because it seems strange to add documentation on `Asset` class that only applies in such specific cases (gsplat asset uncompressed ply load in this case).